.circleci/config.yml

```diff
@@ -312,7 +312,7 @@ jobs:
         command: |
           set -xeo pipefail
 
-          OPERATOR_VERSION=$(python ./scripts/operator/get-last-published-operator-version.py)
+          OPERATOR_VERSION=$(python ./scripts/operator/get_last_published_operator_version.py)
```

Comment:
I think, after @ivanstanev has split the `config.yml` in different files, we can't change it directly. Instead you should make changes in the files on the folder `.circleci/config` and run this commands [here](https://github.com/snyk/kubernetes-monitor/blob/staging/.circleci/README.md) to re-generate the `config.yml`.

Reply:
Ah of course, this is the generated file 🤦‍♀️ 👍